.gitignore

```diff
@@ -157,3 +157,5 @@ docker-compose.override.yaml
 ## Notice Folders
 .notice/
 .notice-work/
+.aider*
+.env
```

Comment:
Surprised this one wasn't needed before - I'm guessing it's just also used by Aider?

Reply:
Yes, this is just to ignore aider-related files.